lib/features/family/shared/widgets/inputs/family_search_field.dart

```diff
@@ -29,17 +29,42 @@ class FamilySearchField extends StatelessWidget {
       focusNode: focusNode,
       style: Theme.of(context).textTheme.labelLarge,
       decoration: InputDecoration(
-        /*suffixIcon: GestureDetector(
-          onTap: controller.clear,
-          child: const FaIcon(
-            FontAwesomeIcons.xmark,
-            color: AppTheme.primary30,
+        contentPadding: EdgeInsets.zero,
+        suffixIcon: GestureDetector(
+          onTap: () {
+            controller.clear();
+            onChanged?.call('');
```

Comment:
to be sure; this onchanged?.call requires an empty string here? :)

Reply:
Yes, to explain:

onChanged is defined by flutter TextField itself and the value is String and not String?

When I clear the controller onChanged isn't automatically called by the TextField so I'm doing it manually here.

If I wouldn't, and you would only check the onChanged and not subscribe to the textController, then for example:
1) if you enter some text and the search filters are applied, and then you press backspace until its gone the search filters are cleared
2) if you enter some text and the search filters are applied, and then you press the clear icon, the search filters are still applied

Reply:
Thanks! :)